sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusReceivedMessage.cs

```diff
@@ -26,7 +26,7 @@ public class ServiceBusReceivedMessage
         /// </summary>
         /// <param name="body">The payload of the message represented as bytes.</param>
         internal ServiceBusReceivedMessage(ReadOnlyMemory<byte> body)
-            : this(new AmqpAnnotatedMessage(new AmqpMessageBody(new ReadOnlyMemory<byte>[] { body })))
+            : this(new AmqpAnnotatedMessage(new AmqpMessageBody(Amqp.Body.FromReadOnlyMemorySegments(new ReadOnlyMemory<byte>[] { body }))))
```

Comment:
nit: can we add a using instead of fully qualifying?

Reply:
I would need to use an alias because the class has a `Body` property. `ServiceBusMessage` also uses the fully qualified name because of this reason

Reply:
I think we should rename the Body class. Can we make it `MessageBody`?

Reply:
Sure